pilot/pkg/model/validation.go

```diff
@@ -2127,9 +2127,10 @@ func ValidateServiceEntry(name, namespace string, config proto.Message) (errs er
 		return fmt.Errorf("cannot cast to service entry")
 	}
 
-	if len(serviceEntry.Hosts) == 0 {
-		errs = appendErrors(errs, fmt.Errorf("service entry must have at least one host"))
+	if len(serviceEntry.Hosts) == 0 && len(serviceEntry.Addresses) == 0 {
+		errs = appendErrors(errs, fmt.Errorf("addresses and/or hosts must be specified"))
```

Comment:
how about `either addresses or hosts must be specified`

Reply:
With "either" users may think that they can't have both. That's why I mentioned "and/or". You think it's unclear?

Reply:
make sense